packages/rancher-monitoring/package.yaml

```diff
@@ -2,4 +2,5 @@ url: https://kubernetes-charts.storage.googleapis.com/prometheus-operator-8.16.1
 packageVersion: 00
 generateCRDChart:
   enabled: true
-  providesGVR: prometheuses.monitoring.coreos.com/v1
\ No newline at end of file
+  providesGVR: prometheuses.monitoring.coreos.com/v1
```

Comment:
Provides has moved outside of generateCRDChart

Reply:
Will be updated in the latest commit